sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -595,8 +595,7 @@ public EncryptedBlobClientBuilder containerName(String containerName) {
      * @throws NullPointerException If {@code blobName} is {@code null}
      */
     public EncryptedBlobClientBuilder blobName(String blobName) {
-        this.blobName = Utility.urlEncode(Utility.urlDecode(Objects.requireNonNull(blobName,
-            "'blobName' cannot be null.")));
+        this.blobName = Objects.requireNonNull(blobName, "'blobName' cannot be null.");
```

Comment:
Remove it also from `EncryptedBlobClientBuilder endpoint(String endpoint)`

https://github.com/Azure/azure-sdk-for-java/blob/5f96f6ab2124bb45095670866fb89ff850074cad/sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java#L562